virttest/qemu_vm.py

```diff
@@ -1043,7 +1043,20 @@ def set_value(opt_string, key, fallback=None):
             elif optget("spice_port") != "no":
                 set_value("port=%s", "spice_port")
 
-            set_value("password=%s", "spice_password", "disable-ticketing=on")
+            password = optget("spice_password")
+            secret_cmdline = ''
+            if password:
+                if process.run(
+                        '%s -spice help | grep password-secret' % qemu_binary,
+                        ignore_status=True, shell=True).exit_status:
```

Comment:
Suggest `if "password-secret" not in devices.execute_qemu("-spice help"):`.

Reply:
@yanan-fu Updated this one as you recommended, thanks.